tests/test_ray_finetuning_on_cpu.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
The test name must be `test_finetuning.sh`, otherwise test won't be triggered when there is no test scripts change.